face_track/main.py

```diff
@@ -27,9 +27,9 @@
 
 print "Start face-tracking node"
 ft = FaceTrack()
-
+ft.do_look_at_actions()
 while not rospy.is_shutdown():
-	ft.do_look_at_actions()
+	#ft.do_look_at_actions()
```

Comment:
Yeah, now that you mention it, I guess they use lots of async i/o in ros. I guess they had to do something like that, because python internals are kind-of wacky.